### PR TITLE
EOS-25051:S3:try changing timeout value of motr reconnect 

### PR DESCRIPTION
**Pull request by sachitanands**
Signed-off-by: Sachitanand Shelake <sachitanand.shelake@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: server/s3server.cc [hunk 1]
```diff
@@ -943,7 +943,7 @@ int main(int argc, char **argv) {
 
     struct timeval tv;
     tv.tv_usec = 0;
-    tv.tv_sec = MOTR_INIT_MAX_ALLOWED_TIME;
+    tv.tv_sec = sleep_time;
```
Comment (nileshgovande) [inline server/s3server.cc:946]: Modify MOTR_INIT_MAX_ALLOWED_TIME to 90 and we can merge it

Comment (nileshgovande) [reply server/s3server.cc:946]: as a workaround

File: server/s3server.cc [hunk 2]
```diff
@@ -60,7 +60,7 @@
 #define MIN_RESERVE_SIZE 32768
 
 #define WEBSTORE "/home/seagate/webstore"
-#define MOTR_INIT_MAX_ALLOWED_TIME 15
```
Comment (nileshgovande) [inline server/s3server.cc:63]: change to 60

Comment (sachitanands) [reply server/s3server.cc:63]: done

**Review by yeshpal-jain-seagate — APPROVED** [review]

**Pull request closed without merging**